beacon_node/network/src/sync/network_context.rs

```diff
@@ -455,7 +459,6 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
                 (None, None)
             };
 
-        // TODO(pawan): this would break if a batch contains multiple epochs
```

Comment:
Even if a batch contains multiple epochs (which it doesn't), this is a very niche case that would only break during fork boundaries.